app/{vmagent,vminsert}: adds /v1/metrics suffix for opentelemetry route path (#5871)
VictoriaMetrics/VictoriaMetrics: docs/CHANGELOG.md

* FEATURE: [vmalert](https://docs.victoriametrics.com/#vmalert): support filtering by group, rule or labels in [vmalert's UI](https://docs.victoriametrics.com/vmalert/#web) for `/groups` and `/alerts` pages. See [the pull request](https://github.com/VictoriaMetrics/VictoriaMetrics/pull/5791) by @victoramsantos.

* BUGFIX: all VictoriaMetrics components: return back periodic closing of incoming connections to `-httpListenAddr` every 2 minutes, which was disabled in [v1.98.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.98.0) when addressing [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1304#issuecomment-1636997037). See [this comment](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/1304#issuecomment-1961891450) for details on why the periodic closing of incoming connections has been returned back.

> * BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent.html): adds missing `/v1/metrics` suffix for `/opentelemetry/api/v1/push/v1/metrics` opentelemetry ingestion path. This suffix is hardcoded at protocol [specification](https://opentelemetry.io/docs/specs/otlp/#otlphttp-request)

Reviewer: ```suggestion
* BUGFIX: [vmagent](https://docs.victoriametrics.com/vmagent.html): accept OpenTelemetry data at `/opentelemetry/v1/metrics`, since the `/v1/metrics` suffix is hardcoded at [OpenTelemetry protocol specification](https://opentelemetry.io/docs/specs/otlp/#otlphttp-request).
```
